packages/blocks/src/edgeless-text/edgeless-text-service.ts

```diff
@@ -54,10 +54,11 @@ export class EdgelessTextBlockService extends BlockService<EdgelessTextBlockMode
             const abortController = new AbortController();
             edgelessText.addEventListener(
               'focusout',
-              () => {
+              e => {
                 if (
                   !paragraph.model.text ||
-                  paragraph.model.text.length === 0
+                  (paragraph.model.text.length === 0 &&
+                    e.relatedTarget !== null)
```

Comment:
If the bookmark input modal is closed without any content, the empty `edgeless-text` will be retained.

Reply:
Compared to the complete inability to add the bookmark before the fix, the issue you pointed out is still acceptable, so I would like to merge this fix first. Additionally, I feel that the edgeless text is limited in this scenario because it does not restrict the interaction methods of the internal Block.